site/docs/supported-models/_components/text-rerank-models-table/models.ts

```diff
@@ -45,6 +49,7 @@ export const TEXT_RERANK_MODELS: TextRerankModelType[] = [
   },
   {
     architecture: 'Qwen3ForCausalLM',
+    optimumIntelTask: 'text-generation-with-past',
```

Comment:
that is strange. How do you use `hidden states` or `output_tokens`? You need scores for reranking.

Reply:
Thanks, I got it. From my perspective we need to adjust it on optimum-intel side. That means we need to have classification task for it where we should prepend this post-processing on optimum-intel side so that IR will contain all required ops and no need to prepend it on GenAI side. That's for the future, but will be nice to have JIRA ticket for such enhancement. How do you think?

Reply:
The post-processing applied to the model depends on the tokenizer's padding side. We can sync this in GenAI's embedding pipeline, but the optimum class does not have a direct link to the tokenizer.